lmeval-scenarios.adoc

```diff
@@ -0,0 +1,310 @@
+
+
+=== Managed PVCs
+
+To enable a managed PVC simply specify its size. This creates a PVC named `<job-name>-pvc` (in this case `evaljob-sample-pvc`) which will be available after the job finishes, but is deleted when the LMEvalJob is deleted.
+
+.Procedure
+* Enter the following code:
++
+[source]
+----
+apiVersion: trustyai.opendatahub.io/v1alpha1
+kind: LMEvalJob
+metadata:
+  name: evaljob-sample
+spec:
+  # other fields omitted ...
+  outputs: 
+    pvcManaged: 
+      size: 5Gi 
+----
+
+.Notes on the code
+* `outputs` is the section for specifying custom storage locations
+* `pvcManaged` will create an operator-managed PVC
+* `size` (compatible with standard PVC syntax) is the only supported value
+
+
+=== Existing PVCs
+
+To use an already existing PVC you can pass its name as a reference. The PVC must already exist when the LMEvalJob is created. 
+In this case, the PVC is not managed by the TrustyAI operator, so it will be available even after deleting the LMEvalJob.
+
+
+.Procedure
+* Create a PVC. An example is the following.
++
+[source]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "my-pvc"
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+----
+* Reference this PVC from the LMEvalJob (note that `pvcName` references the already existing PVC `my-pvc`)
++
+[source]
+----
+apiVersion: trustyai.opendatahub.io/v1alpha1
+kind: LMEvalJob
+metadata:
+  name: evaljob-sample
+spec:
+  # other fields omitted ...
+  outputs:
+    pvcName: "my-pvc" 
+----
+
+
+
+== Using an InferenceService
+
+This example assumes that the vLLM model is already deployed in your cluster.
+.Prerequisites
+* You have logged in to Red Hat OpenShift AI.
+
+* Your OpenShift cluster administrator has installed OpenShift AI and enabled the TrustyAI service for the data science project where the models are deployed.
+
+.Procedure
+* Define your LMEvalJob CR:
+[source]
+----
+  apiVersion: trustyai.opendatahub.io/v1alpha1
+kind: LMEvalJob
+metadata:
+  name: evaljob
+spec:
+  model: local-completions
+  taskList:
+    taskNames:
+      - mmlu
+  logSamples: true
+  batchSize: 1
+  modelArgs:
+    - name: model
+      value: granite
+    - name: base_url
+      value: $ROUTE_TO_MODEL/v1/completions 
+    - name: num_concurrent
+      value:  "1"
+    - name: max_retries
+      value:  "3"
+    - name: tokenized_requests
+      value: "False"
+    - name: tokenizer
+      value: ibm-granite/granite-7b-instruct
+ env:
+   - name: OPENAI_TOKEN
+     valueFrom:
+          secretKeyRef: 
+            name: <secret-name> 
+            key: token 
+----
+
+* Apply this CR into the same namespace as your model. You should see a pod spin up in your model namespace called `evaljob`. In the pod terminal, you can see the output via `tail -f output/stderr.log`
+
+.Notes on the code
```

Comment:
Same note as previous - use code callouts immediately after the code sample (line 297)